package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.12.1",
   "description": "An advanced barcode-scanner written in JavaScript",
   "main": "lib/quagga.js",
+  "types": "./type-definitions/quagga.d.ts",
   "browser": "dist/quagga.min.js",
```

Comment:
why do you put `./` at the start? browser and main fields do not have same, why you add it there?

Reply:
yes, should be without the `./` for consistency